lib/view/backupwallet/backup_wallet_index.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:wallet_app/view/backupwallet/backup_wallet_words.dart';
+import 'package:wallet_app/view/main_view/main_page.dart';
+
+class BackupWalletIndex extends StatelessWidget {
+
+
+
+  Widget buildDialogWindow(BuildContext context){
+    return SimpleDialog(
+      contentPadding: const EdgeInsets.all(0.0),
+      children: <Widget>[
+        Center(
+          child:Column(
+            children: <Widget>[
+              Padding(
+                padding: const EdgeInsets.only(top: 10),
+                child: Icon(
+                  Icons.camera_enhance,
+                  size: 60,
+                ),
+
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 5),
+                child: Text(
+                    WalletLocalizations.of(context).backup_index_prompt_title,
+                  style: TextStyle(
+                    fontSize: 24,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(left: 12,right: 12,top: 10,bottom: 20),
+                child: Text(WalletLocalizations.of(context).backup_index_prompt_tips,textAlign: TextAlign.center,),
+              ),
+              InkWell(
+                radius: MediaQuery.of(context).size.width*0.5,
+                splashColor:Colors.blue,
+                onTap: (){
+                  Navigator.of(context).pop();
+                  Navigator.push(context, MaterialPageRoute(builder: (BuildContext context) => BackupWalletWords()));
+                },
+                child: Container(
+                  child: Padding(
+                    padding: const EdgeInsets.only(top: 10,bottom: 10),
+                    child: Text(WalletLocalizations.of(context).backup_index_prompt_btn,textAlign: TextAlign.center,
+                      style: TextStyle(
+                        fontSize: 20,
+                        color: Colors.white
+                    ),),
+                  ),
+                  width: double.infinity,
+                  decoration: BoxDecoration(color: Colors.red),
+                ),
+              ),
+            ],
+          )
+        )
+      ],
+    );
+  }
+  void onTouchBtn(BuildContext context){
+    showDialog(
+      context: context,
+      builder:buildDialogWindow,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    Widget pageContent(){
+      return Column(
+        children: <Widget>[
+          Container(
+            margin: EdgeInsets.only(top: 30,left: 20,right: 20),
+            child: CachedNetworkImage(
+              placeholder: (context, url) => new CircularProgressIndicator(),
+              errorWidget: (context, url, error) => new Icon(Icons.error),
+              imageUrl: 'http://www.pptbz.com/pptpic/UploadFiles_6909/201203/2012031220134655.jpg',
```

Comment:
This url does not resolve, is it supposed to?
If so, can you double check or include the asset locally?

Reply:
Fixed.